### PR TITLE
Removing member spotlight

### LINKED ISSUE #30: Do we want to take out the member spotlight?
Since It's kinda difficult to keep up with every month. It was fun to throw in when I was revamping it, but I'm not sure if it's good idea anymore. I can write the pr if everyone agrees.

### PR DESCRIPTION
**Pull request by nslee333**
Resolves #30.

Changes made:
- Added changes to next-env.d.ts and tsconfig added by `npm run dev`
- Removed Member Spotlight from display.
- Removed member content related to myself.
- Left MemberSpotlight component intact for future use.
- Added Title for page.

### CODE REVIEW

File: src/app/page.tsx
Comment (CalebJamesStevens) [inline src/app/page.tsx]: titles go in the head

Comment (nslee333) [reply src/app/page.tsx]: Can you elaborate please? Just need to know where it needs to go.

Comment (CalebJamesStevens) [reply src/app/page.tsx]: yes sorry, title tags are meta data and therefor go in the head and not the body of the html document

In next13 this is handled with a metadata export

https://nextjs.org/docs/app/building-your-application/optimizing/metadata

**Review by ctsstc — APPROVED** [review]
This looks good to me, thanks! I'll let someone else validate and merge this in, unless this sits for too long.

Edit: Looks good here: https://deploy-preview-31--friendly-dubinsky-eae4c4.netlify.app/

**Review by CalebJamesStevens — APPROVED** [review]

**Pull request merged**
